### ISSUE TITLE
How to reproduce the result PQ 59.8

### ISSUE THREAD

**Issue by LetsGoFir** [bowenc0221/panoptic-deeplab#10]
Thanks for your great work!

Which one config file to choose to reproduce the PQ 59.8? 
I use the 'configs/panoptic_deeplab_R50_os32_cityscapes.yaml' and get PQ 58.3.
BTW, before you change the head dimension from 128 to 32, I got PQ 59.1.

**Comment by bowenc0221** [comment]
Yes, you are using the right config.
I found there is a very high variance if you train the model with batchsize 4.
I would suggest you either train the model multiple times with batchsize 4 or set the batchsize to 8 (if resource allows).

**Comment by LetsGoFir** [comment]
> Yes, you are using the right config.
> I found there is a very high variance if you train the model with batchsize 4.
> I would suggest you either train the model multiple times with batchsize 4 or set the batchsize to 8 (if resource allows).

Thanks for your prompt reply, should the learning rate be changed when using batch size 8?

**Comment by bowenc0221** [comment]
No, you don't need to change anything except the batchsize.
